hvplot/tests/testibis.py

```diff
@@ -0,0 +1,73 @@
+"""Ibis works with hvplot"""
+import sqlite3
+from pathlib import Path
+
+import holoviews as hv
+import matplotlib
+import pytest
+
+matplotlib.use('agg')
+
+try:
+    import duckdb
```

Comment:
Does duckdb get installed with ibis?

Reply:
only if you `pip install ibis-framework[duckdb]`